google-cloud-spanner-hibernate-testing/src/test/java/org/hibernate/userguide/hql/HQLTest.java

```diff
@@ -2041,22 +1947,6 @@ public void test_hql_like_predicate_example_2() {
 		});
 	}
 
-	@Test
-	public void test_hql_like_predicate_escape_example() {
-
-		doInJPA( this::entityManagerFactory, entityManager -> {
-			//tag::hql-like-predicate-escape-example[]
-			// find any person with a name starting with "Dr_"
-			List<Person> persons = entityManager.createQuery(
-				"select p " +
-				"from Person p " +
-				"where p.name like 'Dr|_%' escape '|'", Person.class )
```

Comment:
this tests `LIKE` ?  That is in Cloud Spanner though

Reply:
This one doesn't work because it uses `like` with `escape` to specify a custom character to use as an escape delimiter. The escape half is not yet supported in Spanner.